lib/commands/wrappers/schedule.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Commands
+  module Wrappers
+    class Schedule < Base
+      def self.call(schedules:)
+        schedules.map do |schedule|
+          data = [
+            "#{schedule.time_icon} #{schedule.starts_at.strftime('%H:%M')}",
+            "#{schedule.event_icon} #{schedule.title}"
+          ]
+
+          data.push(["#{schedule.speaker_icon} #{schedule.speaker.full_name}"]) unless schedule.speaker.nil?
+
+          data.join("\n")
```

Comment:
I guess will be better if all code from the block will be moved to another class-formatter. For example like that:

```ruby
schedules.map do |schedule|
  Schedule::ItemFormatter.call(schedule)
end.join("\n\n")
```

```ruby
module Shedule
  class ItemFormatter
     # . . .

     def call
        [time_label, title, speaker_info].compact.join("\n")
     end

    private

    def time_label
       "#{schedule.time_icon} #{schedule.starts_at.strftime('%H:%M')}"
    end

    def title
      "#{schedule.event_icon} #{schedule.title}"
    end

    def speaker_info
      return if schedule.speaker.nil?
      "#{schedule.speaker_icon} #{schedule.speaker.full_name}"
    end
  end
end
```

Reply:
Good idea, thank you. I added changes, please review it